modules/apps/frontend-css/frontend-css-variables-web/src/main/java/com/liferay/frontend/css/variables/web/internal/theme/ThemeBundleInspector.java

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.css.variables.web.internal.theme;
+
+import com.liferay.frontend.css.variables.CSSVariableDescription;
+import com.liferay.frontend.css.variables.CSSVariableType;
+import com.liferay.frontend.css.variables.web.internal.CSSVariableDescriptionImpl;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONException;
+import com.liferay.portal.kernel.json.JSONFactory;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.util.LocaleUtil;
+import com.liferay.portal.kernel.util.StringUtil;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import java.net.URL;
+
+import java.util.Dictionary;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import org.osgi.framework.Bundle;
+
+/**
+ * @author Iván Zaera Avellón
+ */
+public class ThemeBundleInspector {
+
+	public static boolean isTheme(Bundle bundle) {
```

Comment:
There seems to exist `pluginAutoDeployListenerHelper.isThemePlugin()`... 

```java
public boolean isThemePlugin() throws AutoDeployException {
	if (isMatchingFile("WEB-INF/liferay-look-and-feel.xml", false)) {
		return true;
	}

	Matcher matcher = _themePluginPattern.matcher(_file.getName());

	if (matcher.find() &&
		isMatchingFile(
			"WEB-INF/liferay-plugin-package.properties", false)) {

		return true;
	}

	return false;
}
```

Could we use it or at least have the same implementation?

Reply:
Yes.